cpu/stm32_common/dist/clk_conf/clk_conf.c

```diff
@@ -0,0 +1,692 @@
+/*
```

Comment:
as the computation in this file is specific for the F2/4/7, I think the file name should reflect that somehow. Or do we plan to put all STM32 CPUs computation in this file?

Reply:
In #7500 I added support for F0 and F1